src/pytest_replay/__init__.py

```diff
@@ -100,3 +129,7 @@ def pytest_configure(config):
 def pytest_report_header(config):
     if config.getoption("replay_record_dir"):
         return "replay dir: {}".format(config.getoption("replay_record_dir"))
+
+
+def pytest_cmdline_main(config):
+    config.option.replay_start_time = time.time()
```

Comment:
As far as I know, if we do not use this approach and let the ``replay_start_time`` as a class attribute or anything like that which directly relies on the ``ReplayPlugin``. We may end up with different start times for each worker.
With that in mind, I thought to use the ``pytest_cmdline_main`` to add an option with the ``replay_start_time`` which will follow this information for each worker using the ``ReplayPlugin``, in that way, we will have consistency with the start time.

But please feel free to point a better approach! :)

Reply:
Cool, but there's a better way indeed, we can implement `pytest_configure_node` to pass information to a a node:

```python
class DeferPlugin:

    def pytest_configure_node(self, node):
        node.config.workerinput['replay_start_time'] = node.config.replay_start_time

def pytest_configure(config):
    if hasattr(config, 'workerinput'):
        # worker node
        config.replay_start_time = config.workerinput['replay_start_time']
    else:
        # master node 
        config.replay_start_time = datetime.now() 
```

Problem is that `pytest_configure_node` needs to be declared lazily in case xdist is not installed: https://docs.pytest.org/en/latest/writing_plugins.html#optionally-using-hooks-from-3rd-party-plugins

Reply:
Thanks for pointing me a better way! :)
I will update it soon